testing/helpers/index.js

```diff
@@ -62,3 +62,54 @@ export async function awaitFrameAfter(ms) {
   await sleep(ms);
   await afterRenderPromise();
 }
+
+const VOID_ELEMENTS = new Set([
```

Comment:
What does "void element" mean?

Reply:
I think they are now named "Empty Elements": https://developer.mozilla.org/en-US/docs/Glossary/Empty_element.

Reply:
```suggestion
// See https://developer.mozilla.org/en-US/docs/Glossary/Empty_element
const VOID_ELEMENTS = new Set([
```

Reply:
Hah! I have this in the PR I just haven't pushed yet 😄